subsys/bluetooth/controller/Kconfig

```diff
@@ -16,12 +16,20 @@ if BT_CTLR
 
 choice BT_LL_CHOICE
 	prompt "Bluetooth Link Layer Selection"
-	default BT_LL_SW
 	help
 	  Select the Bluetooth Link Layer to compile.
 
+config BT_LL_SW_SPLIT
+	bool "Software-based BLE Link Layer (ULL/LLL split)"
```

Comment:
Rename this perhaps?
Most people don't know what ULL/LLL split is, or care about that.

Reply:
@joerchan suggestion?

Reply:
I would suggest to rename this to BT_LL_SW and the old one to BT_LL_SW_LEGACY
Also remove the (ULL/LLL split) part of the description, and put (Legacy) in the description of the old one.

Reply:
I agree that is the better option too @cvinayak. That way the switch is seamless, although we do slightly change the meaning of a Kconfig option.

Reply:
Just to join the bikeshedding: That would be a bit annoying, as we would need to change the places where today we use BT_LL_SW_SPLIT. Anyhow, chose whichever you guys prefer

Reply:
How about BT_LL_SW --> BT_LL_SW_LEGACY and keep the BT_LL_SW_SPLIT as is? in a subsequent PR, BT_LL_SW_SPLIT --> BT_LL_SW can be done.